pkg/storage/export/export/export.go

```diff
@@ -510,6 +518,14 @@ func (ctrl *VMExportController) checkPod(vmExport *exportv1.VirtualMachineExport
 		return nil
 	}
 
+	if ttlExpiration := getExpirationTime(vmExport); !time.Now().Before(ttlExpiration) {
+		if err := ctrl.Client.VirtualMachineExport(vmExport.Namespace).Delete(context.Background(), vmExport.Name, metav1.DeleteOptions{}); err != nil {
+			return err
+		}
+		vmExport.SetDeletionTimestamp(&metav1.Time{Time: time.Now()})
```

Comment:
this seems like really bad form to me.  Should only be set by k8s apiserver

Reply:
why is it done?

Reply:
So we don't attempt status update on a deleted export
https://github.com/kubevirt/kubevirt/blob/c4452b8643a39af7c4a51223a988bf0bd71f78fb/pkg/storage/export/export/export.go#L464

Reply:
>  Worst case, I guess we try to update the resource after it is gone and get an error

Yep, that is exactly the approach I took with the current state of the PR